packages/py3_unsupported_python.rb

```diff
@@ -1,16 +1,17 @@
 require 'buildsystems/pip'
 
 class Py3_unsupported_python < Pip
-  description 'This package can be used as a conditional dependency to indicate lack of support for particular versions of Python.'
```

Comment:
where did the description go?

Reply:
Actually, how did this get past CI? This is removing a description, which should cause `tests/prop_test` to fail.